### PR TITLE
Feature/lecture

### PR DESCRIPTION
**Pull request by gorilla-Kim**

### CODE REVIEW

File: src/components/Modal/PDF.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Modal, Button } from 'antd';
+import Lecture from "../PDF/Lecture";
+import styled from "styled-components";
+
+const PDF = ({title, PDFurl}) => {
+  const [visible, setVisible] = useState(false);
+  console.log(PDFurl)
```
Comment (FlaSh0211) [inline src/components/Modal/PDF.js]: delete consolelog

Comment (gorilla-Kim) [reply src/components/Modal/PDF.js]: remove console log complete!
thank you friend~

File: src/components/PDF/Lecture.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    })
+  }
+  const prev = () => {
+    setPageNumber(pageNumber => {
+      if(pageNumber - 1 <= 0) {
+        return 1
+      }
+      return pageNumber - 1
+    })
+  }
```
Comment (gorilla-Kim) [inline src/components/PDF/Lecture.js]: 삼항연산자를 쓰는게 더 좋아보임

`
pageNumber + 1 >= numPages ? numPages : pageNumber+1
`

Comment (gorilla-Kim) [reply src/components/PDF/Lecture.js]: @FlaSh0211  어떤가요

**Review by FlaSh0211 — APPROVED** [review]

**Pull request merged**